.github/workflows/ci.yml

```diff
@@ -161,7 +161,12 @@ jobs:
     steps:
       - uses: taiki-e/checkout-action@v1
       - name: Install Rust
-        run: rustup toolchain add nightly --no-self-update && rustup default nightly
+        run: rustup toolchain add stable --no-self-update --profile minimal && rustup default stable
+      - name: Generate Cargo.lock
+        run: cargo update
```

Comment:
Does this make the cache invalid every time a dependency is updated?

Reply:
Yes, on every patch release it is invalidated, though it would still reuse the previous cache.

If the rustc version is changed, then it would start afresh.

Reply:
BTW it will also start afresh if `RUSTFLAGS` or any `CARGO*` env has changed.